### PR TITLE
csomh 👋 brb

### PR DESCRIPTION
**Pull request by csomh**

### CODE REVIEW

File: clone_roles.py [hunk 1]
```diff
@@ -62,11 +62,21 @@ def get_issue_by_title(title: str, issues: List[Issue]):
...
+        # Some maintainers who were active might not
+        # be in the PEOPLE set already.
+        # Replace them with someone from candidates.
+        # Chose someone randomly if candidates run out.
+        for i, maintainer in enumerate(maintainers):
+            if maintainer not in PEOPLE:
+                try:
+                    maintainers[i] = candidates.pop()
+                except IndexError:
+                    maintainers[i] = random.choice(list(PEOPLE))
```
Comment (mfocko) [inline clone_roles.py:79]:
```suggestion
                maintainers[i] = candidates.pop() if candidates else random.choice(list(PEOPLE))
```

**Review by mfocko — APPROVED** [review]

File: clone_roles.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 PEOPLE = {
     "lachmanfrantisek",
     "FrNecas",
-    "csomh",
+    # "csomh",
```
Comment (mfocko) [inline clone_roles.py:13]: :eyes:

**Review by FrNecas — APPROVED** [review]

File: clone_roles.py (hunk 1 above)
Comment (csomh) [reply clone_roles.py]: I couldn't resist and factored this out as `next_candidate()`, and use it instead of the previous `candidates.pop()`, too.

**Review by mfocko — APPROVED** [review]
:pinched_fingers:

**Pull request merged**